clang/test/SemaSYCL/esimd-private-global.cpp

```diff
@@ -1,5 +1,15 @@
 // RUN: %clang_cc1 -fsycl-is-device -fsycl-explicit-simd -fsyntax-only -verify -pedantic %s
 
```

Comment:
Please add a comment here briefly describing what the test does.

Reply:
Done